usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -1291,10 +1299,11 @@
                 ("prime_award_object_classes_funding_this_award", None),  # Annotation is used to create this column
                 ("prime_award_program_activities_funding_this_award", None),  # Annotation is used to create this column
                 ("prime_awardee_duns", "broker_subaward__awardee_or_recipient_uniqu"),
+                ("prime_awardee_uei", "award__latest_transaction__contract_data__awardee_or_recipient_uei"),
                 ("prime_awardee_name", "broker_subaward__awardee_or_recipient_legal"),
                 ("prime_awardee_dba_name", "broker_subaward__dba_name"),
                 ("prime_awardee_parent_duns", "broker_subaward__ultimate_parent_unique_ide"),
-                ("prime_awardee_parent_name", "broker_subaward__ultimate_parent_legal_enti"),
```

Comment:
Why remove this field from the download?

Reply:
sometimes we make mistakes